package/MDAnalysis/analysis/lineardensity.py

```diff
@@ -50,24 +50,58 @@ class LinearDensity(AnalysisBase):
           histograms. Defines the resolution of the resulting density
           profile (smaller --> higher resolution)
     verbose : bool (optional)
-          Show detailed progress of the calculation if set to ``True``; the
-          default is ``False``.
+          Show detailed progress of the calculation if set to ``True``
 
     Attributes
     ----------
-    results : dict
-          Keys 'x', 'y', and 'z' for the three directions. Under these
-          keys, find 'pos', 'pos_std' (mass-weighted density and
-          standard deviation), 'char', 'char_std' (charge density and
-          its standard deviation), 'slice_volume' (volume of bin).
+    results.x.dim : int
+           index of the x axes (0)
+    results.x.pos : numpy.ndarray
+           mass density in x direction
+    results.x.pos_std : numpy.ndarray
+           standard deviation of the mass density in x direction
+    results.x.char : numpy.ndarray
+           charge density in x direction
+    results.x.char_std : numpy.ndarray
+           standard deviation of the charge density in x direction
+    results.x.slice_volume : float
+           volume of bin in x direction
+    results.y.dim : int
+           index of the y axes (1)
+    results.y.pos : numpy.ndarray
+           mass density in y direction
+    results.y.pos_std : numpy.ndarray
+           standard deviation of the mass density in y direction
+    results.y.char : numpy.ndarray
+           charge density in y direction
+    results.y.char_std : numpy.ndarray
+           standard deviation of the charge density in y direction
+    results.y.slice_volume : float
+           volume of bin in y direction
+    results.z.dim : int
+           index of the z axes (2)
+    results.z.pos : numpy.ndarray
+           mass density in z direction
+    results.z.pos_std : numpy.ndarray
+           standard deviation of the mass density in z direction
+    results.z.char : numpy.ndarray
+           charge density in z direction
+    results.z.char_std : numpy.ndarray
+           standard deviation of the charge density in z direction
+    results.z.slice_volume : float
+           volume of bin in z direction
 
     Example
     -------
-    First create a LinearDensity object by supplying a selection,
-    then use the :meth:`run` method::
+    First create a `LinearDensity` object by supplying a selection,
```

Comment:
```suggestion
    First create a ``LinearDensity`` object by supplying a selection,
```

Use double backticks for fun monospace in rendered docs -- single backticks if followed by a sphinx directive like `:meth:`. I believe single backticks italicise instead.